### PR TITLE
Added github workflow to exec. unit tests

### PR DESCRIPTION
**Pull request by alexbjorlig**
The workflow does not include notification of gitter, but if that is still needed there is a pluglin like this one [here](https://github.com/marketplace/actions/gitter-notify).

### CODE REVIEW

File: .github/unit-tests.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: bahmutov/npm-install@v1
+      - run: npm test
+      - run: npm run coverage 
+ # Maybe we should add https://github.com/marketplace/actions/gitter-notify here?
```
Comment (addaleax) [inline .github/unit-tests.yml:25]: I don’t think we need that, no :)

File: .github/unit-tests.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
```
Comment (addaleax) [inline .github/unit-tests.yml:19]:
```suggestion
        uses: actions/setup-node@v2
```

File: .github/unit-tests.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: bahmutov/npm-install@v1
```
Comment (addaleax) [inline .github/unit-tests.yml:22]: We don’t need this line unless we want a specific npm version, do we?

File: .github/unit-tests.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+name: unit tests
+
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (addaleax) [inline .github/unit-tests.yml:6]:
```suggestion
on: [push, pull_request]
```

**Review by addaleax — APPROVED** [review]

File: .github/unit-tests.yml (hunk 3 above)
Comment (alexbjorlig) [reply .github/unit-tests.yml:22]: `bahmutov/npminstall` is just for installing dependencies with caching --> works really great. A will just push a new commit with suggested changes

**Comment by addaleax** [comment]
Ok, had to make a small change to the location of the file but seems to be working now :)

**Pull request merged**